net.crossfireatlas.crossfire.appdata.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
You need to explicitly reference this file in your manifest and install it at correct place.